SPR-5503 - Fix get message from resource of basename with dots and refactor license ResourceBundleMessageSourceTests

wertklop opened the pull request:
Fix get message from resource of basename with dots and refactor license ResourceBundleMessageSourceTests

Prior to this commit, doesn't get message from ReloadableResourceBundleMessageSource if basename exist dots but slashes, for example "package.name.messages.properties". Following that change, these names are valid for getting messages

Issue: SPR-5503

pivotal-issuemaster commented:
@wertklop Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-framework&pullRequestId=1176)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-projects/spring-framework&pullRequestId=1176) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/faq) for frequently asked questions.

pivotal-issuemaster commented:
@wertklop Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-framework&pullRequestId=1176)!

wertklop committed:
SPR-5503 - Fix get message from resource of basename with dots and refactor license ResourceBundleMessageSourceTests

Prior to this commit, doesn't get message from ReloadableResourceBundleMessageSource if basename exist dots but slashes, for example "package.name.messages.properties". Following that change, these names are valid for getting messages

Issue: SPR-5503

snicoll commented:
thanks for the PR and sorry it took so long to process it. There hasn't been a lot of traction from the community and the related issue has been closed as a result. Besides, the difference of behavior with ResourceBundleMessageSource is actually documented. Thanks anyway.
